modules/core/src/alloy/proto/validation/ProtoIndexTraitValidator.java

```diff
@@ -90,18 +94,54 @@ private List<ValidationEvent> validateShape(Model model, Shape shape) {
 					.collect(Collectors.groupingBy(e -> e.getValue().getNumber(),
 							Collectors.mapping(Map.Entry::getKey, Collectors.toList())));
 
-			return perIndex.entrySet().stream().filter(e -> e.getValue().size() > 1).map(e -> {
+			indexCheck = perIndex.entrySet().stream().filter(e -> e.getValue().size() > 1).map(e -> {
 				final Integer index = e.getKey();
 				final List<String> labels = e.getValue();
 				final String memberNames = labels.stream().collect(Collectors.joining(","));
 				return ValidationEvent
 						.builder().id(DUPLICATED_PROTO_INDEX).message("Proto index " + index
-								+ " is used muliple times in members " + memberNames + " of shape " + shape + ".")
+								+ " is used multiple times in members " + memberNames + " of shape " + shape + ".")
 						.shape(shape).severity(Severity.ERROR).build();
-			}).collect(Collectors.toList());
+			});
 		} else {
-			return Collections.emptyList();
+			indexCheck = Stream.empty();
 		}
+
+		Stream<ValidationEvent> openStringEnumChecks = null;
+		if ((shape.isEnumShape() || shape.isIntEnumShape()) && shape.hasTrait(OpenEnumTrait.class)) {
+			if (noFieldIsIndexed) {
+				openStringEnumChecks = Stream.empty();
+			} else {
+				openStringEnumChecks = Stream.of(ValidationEvent.builder().id(OPEN_ENUM_MUST_NOT_HAVE_INDEXES)
+						.message("Members of enumeration" + shape + "must not carry proto indexes.").shape(shape)
```

Comment:
What do you think of: `must not have protoIndex annotation`?

Reply:
[28f904e](https://github.com/disneystreaming/alloy/pull/129/commits/28f904e489467a6b9afa4256d5bfa5469ad8c5d8)